server/app.js

```diff
@@ -47,12 +47,114 @@ const apiLimiter = rateLimit({
     legacyHeaders: false, // Disable the `X-RateLimit-*` headers
   })
 
+
+  // Define a new route that retrieves data from the MongoDB database and returns it as a response.
+app.get('/api/logs', async (req, res) => {
+  try {
+    const Log = mongoose.model('Log', {
+      ID: String,
+      loginTime: Date,
+      logoutTime: Date,
+    });
+    const logs = await Log.find();
+    res.json(logs);
+  } catch (error) {
+    console.error(error);
+    res.status(500).json({ error: 'Internal server error' });
+  }
+});
+
 // put routes here for now - I have already created user sign up and login look in controllers/ routes and middleware
 // get users
+const Student = mongoose.model('Student', {
+  username: String,
+  _id: mongoose.Schema.Types.ObjectId,
```

Comment:
We already have the model of user, so we need to use either user or student not both. If we are using student the model needs to contain the password for login. I think it will be easier to use student as you have created all of the endpoints and we only have signup and login at the moment. If you could add the password to the model I will change the other routes to use student.